tests/test_plugin.py

```diff
@@ -1104,7 +1106,8 @@ def test_bitcoin_backend(node_factory, bitcoind):
     """
     This tests interaction with the Bitcoin backend, but not specifically bcli
     """
-    l1 = node_factory.get_node(start=False, options={"disable-plugin": "bcli"},
+    l1 = node_factory.get_node(start=False, options=OrderedDict([("unimportant-plugin", "bcli"),
+                                                                 ("disable-plugin", "bcli")]),
```

Comment:
it's weird that you have to specify `unimportant` and `disable`. if you're explicitly `disabling` a plugin, we should automatically mark it as unimportant, as the desire to disable it obviously means "I want this thing off".